pkgs/by-name/op/opkssh/package.nix

```diff
@@ -0,0 +1,29 @@
+{
+  buildGoModule,
+  fetchFromGitHub,
+  lib,
+}:
+let
+  # v0.3.0 (cf. https://github.com/openpubkey/opkssh/tree/v0.3.0)
+  version = "15ab593536c5c52fd648749705691992c1b6131e";
+in
+buildGoModule {
+  pname = "opkssh";
+  inherit version;
+
+  src = fetchFromGitHub {
+    owner = "openpubkey";
+    repo = "opkssh";
+    rev = version;
+    hash = "sha256-RtTo/wj4v+jtJ4xZJD0YunKtxT7zZ1esgJOSEtxnLOg=";
+  };
+
+  vendorHash = "sha256-MK7lEBKMVZv4jbYY2Vf0zYjw7YV+13tB0HkO3tCqzEI=";
+
+  meta = {
+    homepage = "https://github.com/openpubkey/opkssh";
+    description = "OpenPubkeySSH: integrating SSO with SSH (short-lived SSH keys with an OpenID provider).";
```

Comment:
Please take a look at the nixpkgs manual for the description field: https://nixos.org/manual/nixpkgs/unstable/#var-meta-description

Reply:
Deal. Edited.

Reply:
Ope. Yep. Thanks.